warp10/src/main/java/io/warp10/continuum/store/Constants.java

```diff
@@ -376,6 +376,7 @@
   public static final String HTTP_PARAM_SAMPLE = "sample";
   public static final String HTTP_PARAM_PREBOUNDARY = "boundary.pre";
   public static final String HTTP_PARAM_POSTBOUNDARY = "boundary.post";
+  public static final String HTTP_PARAM_NODATA = "nodata";
```

Comment:
I would propose `deletemeta` as HTTP parameter here instead of "nodata". IMO, this would made more sence for a user

Reply:
Renamed 'metaonly', 'deletemeta' could incorrectly lead users to think that 'deleteall' is related.